src/main/g8/project/Projects.scala

```diff
@@ -25,7 +25,7 @@ object Projects extends Build {
     .settings(Testing.settings:_*)
 
   lazy val examples = project.in(file("example"))
-    .dependsOn(core % "compile->compile;test->test")
+    .dependsOn(core)
```

Comment:
Right. That's my understanding as well. Clearly we don't need test->test here (it's a separate test module) and not sure if I want example (or anything other than root) to aggregate test.